src/firejail/main.c

```diff
@@ -1503,7 +1503,7 @@ int main(int argc, char **argv, char **envp) {
 		else if (strncmp(argv[i], "--rlimit-as=", 12) == 0) {
 			cfg.rlimit_as = parse_arg_size(argv[i] + 12);
 			if ( cfg.rlimit_as == 0 ) {
-				perror("Error: given rlimit-as is invalid. use only non-negative numbers and k,m,g suffix for size");
+				perror("Error: invalid rlimit-as. only use positive numbers and k, m or g suffix.");
```

Comment:
It's a capital O (not o) after completing a sentence:
perror("Error: invalid rlimit-as. Only use positive numbers and k, m or g suffix.");
Please change this and we're set for merge.